postgraphile/website/postgraphile/config.mdx

```diff
@@ -661,6 +661,17 @@ such as `role`, and will be applied by Postgres.
 
 :::
 
+:::warning
```

Comment:
This probably renders as "DANGER" right now; we should probably soften that a bit, or make it more useful...

Maybe:

```suggestion
:::warning Don't set `name` unless you need to!
```

Please check this renders correctly and remove the backticks if not.

Reply:
Might be nice to add this to a few of the other [adminitions](https://docusaurus.io/docs/markdown-features/admonitions) in the documentation too, definitely a separate PR for that though!